userapi/src/main/java/fk/prof/userapi/api/ProfileViewCreator.java

```diff
@@ -0,0 +1,48 @@
+package fk.prof.userapi.api;
+
+import com.codahale.metrics.Timer;
+import fk.prof.metrics.ProcessGroupTag;
+import fk.prof.metrics.Util;
+import fk.prof.userapi.cache.Cacheable;
+import fk.prof.userapi.model.AggregatedOnCpuSamples;
+import fk.prof.userapi.model.AggregatedProfileInfo;
+import fk.prof.userapi.model.ProfileView;
+import fk.prof.userapi.model.ProfileViewType;
+import fk.prof.userapi.model.tree.CallTreeView;
+import fk.prof.userapi.model.tree.CalleesTreeView;
+
+/**
+ * Created by gaurav.ashok on 08/08/17.
+ */
+public class ProfileViewCreator {
+
+    private static String callerTreeViewCreationTimerPrefix ="views.calltree.create.timer";
```

Comment:
Earlier comment about MetricName applies here too as well as every place in userapi where we are defining metric name

Reply:
Done.